src/Daemon/SentryWriter.cpp

```diff
@@ -222,8 +237,11 @@ void SentryWriter::onFault(int sig, const std::string & error_message, const Sta
 
 #else
 
-void SentryWriter::initialize(Poco::Util::LayeredConfiguration &) {}
-void SentryWriter::shutdown() {}
-void SentryWriter::onFault(int, const std::string &, const StackTrace &) {}
+void SentryWriter::initializeInstance(Poco::Util::LayeredConfiguration &) {}
+SentryWriter * SentryWriter::getInstance() { return nullptr; }
+
+SentryWriter::SentryWriter(Poco::Util::LayeredConfiguration &) {}
+SentryWriter::~SentryWriter() = default;
+void SentryWriter::onFault(int, const std::string &, const FramePointers &, size_t, size_t) {}
```

Comment:
```suggestion
```

Reply:
This placeholders required when sentry is disabled.

Reply:
Here is an example - https://s3.amazonaws.com/clickhouse-test-reports/61997/8b39cb2c6d08a2e86867e068bef0a8689362634a/fast_test/run.log

```
2024-04-11 17:21:08 /ClickHouse/src/Daemon/SentryWriter.cpp:260:20: error: out-of-line definition of 'onFault' does not match any declaration in 'SentryWriter'
2024-04-11 17:21:08   260 | void SentryWriter::onFault(int, const std::string &, const FramePointers &, size_t, size_t) {}
2024-04-11 17:21:08       |                    ^~~~~~~

```

Reply:
Oh, I totally missed this, you are right.